From ac97bdbcd85c7d362a1977eb37055b386a181457 Mon Sep 17 00:00:00 2001
From: Thomas Bechtold <tbechtold@suse.com>
Date: Thu, 12 Sep 2019 15:30:50 +0200
Subject: [PATCH] Add manila-status to man-pages list

manila-status was missing in the list of man-pages. So when building the
man-pages with:

    sphinx-build -b man doc/source doc/build/man

there is no "manila-status" in doc/build/man.

Change-Id: Ia2065af8d0ab51a0248d53b3474845ab429803d1
---
 doc/source/conf.py | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/doc/source/conf.py b/doc/source/conf.py
index aec8e8ae..67675b5f 100644
--- a/doc/source/conf.py
+++ b/doc/source/conf.py
@@ -156,7 +156,9 @@ modindex_common_prefix = ['manila.']
 
 man_pages = [
     ('cli/manila-manage', 'manila-manage', u'Cloud controller fabric',
-     [u'OpenStack'], 1)
+     [u'OpenStack'], 1),
+    ('cli/manila-status', 'manila-status', u'Cloud controller fabric',
+     [u'OpenStack'], 1),
 ]
 
 # -- Options for HTML output --------------------------------------------------
-- 
2.23.0

